python/arcticdb/adapters/lmdb_storage_adapter.py

```diff
@@ -0,0 +1,52 @@
+import shutil
+from typing import Optional, Any
+from tempfile import mkdtemp
+
+from arcticdb.version_store.helper import add_lmdb_library_to_env
+from arcticc.pb2.storage_pb2 import EnvironmentConfigsMap
+
+from .storage_fixture_api import *
+
+
+class LmdbStorageFixture(StorageFixture):
```

Comment:
I think either none of them should have their own class, or all of them should, and I would incline to the latter. The storage adapeters are quite fundamental classes and opening the file to be confronted with a chunky fixture is quite odd

Reply:
The file was misnamed; there are no storage adapters.

> and more importantly maintaining
I fail to see the reasoning behind this. We will have to maintain these otherwise, none of our tests work?

> distributing test fixtures should be very low down our list
Most self-respecting storage solutions have a way for customers to test against dummy storage. As mentioned above, we will have to keep them working anyway, so why not distribute them as unadvertised/use-at-your-own-risk API?